R/tune_grid.R

```diff
@@ -376,7 +379,9 @@ tune_grid_workflow <- function(workflow,
 
   workflow <- set_workflow(workflow, control)
 
-  resamples <- resamples %>% dplyr::select(-.seed)
+  if (any(names(resamples) == ".seed")) {
+    resamples <- resamples %>% dplyr::select(-.seed)
```

Comment:
```suggestion
    resamples <- dplyr::select(resamples, -.seed)
```
Just to match the style for the other dplyr use here